qunicorn_core/core/device_service.py

```diff
@@ -71,6 +73,7 @@ def get_device_dict(devices: [IBMBackend]) -> dict:
             "name": device.name,
             "num_qubits": -1 if device.name.__contains__("stabilizer") else device.num_qubits,
             "is_simulator": 1 if device.name.__contains__("simulator") else 0,
```

Comment:
Wenn das in json und an allen anderen Locations jetzt true/false ist, sollten wir das hier auch so machen oder nicht?